starlite/exceptions/utils.py

```diff
@@ -0,0 +1,44 @@
+from typing import Any, Dict, List, Optional, Union
+
+from pydantic import BaseModel
+from starlette.exceptions import HTTPException as StarletteHTTPException
+from starlette.status import HTTP_500_INTERNAL_SERVER_ERROR
+
+from starlite.enums import MediaType
+from starlite.response import Response
+
+from .exceptions import HTTPException
+
+__all__ = ["create_exception_response"]
+
+
+class ExceptionResponseContent(BaseModel):
+    detail: Optional[str]
+    extra: Optional[Union[Dict[str, Any], List[Any]]]
+    status_code: int
```

Comment:
You can set the default here to 500

Reply:
Yes good idea, however I had to change the `to_dict()` call from `exclude_defaults=True` to `exclude_none=True`, which adds another slight change in behavior to the exception responses. If anyone is doing `if error_response['extra"] is None` for example, this would be breaking... but "extra" wasn't always in the exception response anyway so I doubt this would be common.